packages/govern-server/src/core/utils/network.ts

```diff
@@ -1,13 +1,21 @@
 import { ErrorInvalidNetwork } from '../errors'
-import { NETWORKS } from '../params'
```

Comment:
What do you think about the ``Network`` object I thought to create out of those functions in the first PR?